testv2/tests.yml

```diff
@@ -0,0 +1,102 @@
+- infrastructure: aws_defaults
```

Comment:
I think it might be useful to extend this to specify if the scenario is optional or not.

Reply:
cool idea!